### DISCUSSION TITLE
Imperial distance when navigating/staking out

### DISCUSSION THREAD

**Discussion by chuff26** [opengisch/QField#4850] [Ideas]
So I'm a newb, I'm sorry. Anyway, liking some features of qfield but would be awesome if when selecting a point for navigation it would have imperial measurements as an option. I have the project set to feet and acres but that only seems to effect the measurement tool, please feel free to correct me if I'm wrong. And thank you for any help

**Comment by ZsanettMed** [comment]
Hi @chuff26, 

QField currently does not support the use of imperial measurements in navigation. However, we highly value user feedback as it can lead to future updates and improvements. I've posted your suggestion in our [QField Ideas Platform](https://ideas.qfield.org/app-feature-requests/p/imperial-measurements-for-navigationstaking-out) where you and others can upvote it. Alternatively, you could also consider sponsoring the development of this feature. If you're interested, please drop us a mail at [sales@qfield.com](mailto:sales@qfield.com).

Best regards
Zsanett
